parquet/src/encryption/encrypt.rs

```diff
@@ -74,6 +74,24 @@ impl FileEncryptionProperties {
         self.footer_key.key_metadata.as_ref()
     }
 
+    /// Retrieval of key used for encryption of footer and (possibly) columns
+    pub fn footer_key(&self) -> &Vec<u8> { self.footer_key.key.as_ref() }
+
+    /// Get the column names, keys, and metadata used in column_keys
+    pub fn column_keys(&self) -> (Vec<String>, Vec<Vec<u8>>, Vec<Vec<u8>>) {
+        let mut column_names: Vec<String> = Vec::new();
+        let mut keys: Vec<Vec<u8>> = Vec::new();
+        let mut meta: Vec<Vec<u8>> = Vec::new();
+        for (key, value) in self.column_keys.iter() {
+            column_names.push(key.clone());
+            keys.push(value.key.clone());
+            if let Some(metadata) = value.key_metadata.as_ref() {
+                meta.push(metadata.clone());
+            }
+        }
+        (column_names, keys, meta)
+    }
+
```

Comment:
I don't love the asymmetry here vs FileDecryptionProperties. But, I wasn't sure what else to do.